### ISSUE TITLE
Selected Menu Item does not have "Selected" information in Northstar Menu

### ISSUE THREAD

**Issue by davyrwoo** [microsoft/fluentui#14537]
### Environment Information

- **Package version(s)**: @fluentui/react-northstar@0.51.0
- **Browser and OS versions**: Chrome

### Describe the issue:

Selected menu item does not have aria-selected information, or have any narration that the item is selected. Using screenreaders such as NVDA, there is no narration that the selected menu item is selected.

### Please provide a reproduction of the issue in a codepen:

https://codesandbox.io/s/o5ql0?module=/example.js

#### Actual behavior:

1. Turn on NVDA
2. Focus on the selected item
3. Screen reader reads out only the menu item name

#### Expected behavior:

1. Turn on NVDA
2. Focus on the selected item
3. Screen reader should read menu item name and that it is "selected"

**Comment by khmakoto** [comment]
@jurokapsiar fyi

**Comment by jurokapsiar** [comment]
did you mean to use tabListBehavior to create the tab list? Menu/MenuBar do not really have a concept of selection (see https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html for exmple)

**Comment by davyrwoo** [comment]
@jurokapsiar even with the tabListBehavior, NVDA does not say if the menu item is selected or not.

**Comment by YuanboXue-Amber** [comment]
@davyrwoo what do you think if I focus on fixing this:
- When using `accessibility={tabListBehavior}` for `Menu`, the selected/active MenuItem is having `aria-selected="false"`, while it should have `aria-selected="true"`. 
Here's a [Codesandbox](https://codesandbox.io/s/ancient-butterfly-129hr?file=/example.js:374-405) modified based on your example

Because as jurokapsiar wrote, there's no `aria-selected` mentioned for Menu from aria practices. So I'm not so sure about adding `aria-selected` in `menuBehavior` (the default value for Menu's accessibility prop)

**Comment by davyrwoo** [comment]
@YuanboXue-Amber yes that works
